src/com/company/Main.java

```diff
@@ -3,6 +3,38 @@
 public class Main {
 
     public static void main(String[] args) {
-	// write your code here
+        int i = 1;
+        while (i <= 10)
+        {
```

Comment:
открывающие скобки следует писать на той же строке, где объявляем метод/оператор
```suggestion
        while (i <= 10) {
```